hs_core/views/resource_rest_api.py

```diff
@@ -768,8 +768,10 @@ def list(self, request, *args, **kwargs):
                 try:
                     resource_file_info_list.append(self.resourceFileToListItem(f))
                 except SessionException as err:
-                    # primarily this exception will be raised if the file is not found in iRODS
                     logger.error(f"Error for file {f.storage_path} from iRODS: {err.stderr}")
+                except Exception as err:
+                    # primarily this exception will be raised if the file is not found in iRODS
+                    logger.error(f"Error for file {f.storage_path} from iRODS: {str(err)}")
```

Comment:
maybe just combine both `except` blocks into a single one? Unless I'm forgetting that `SessionException.stderr` is significantly different than simply `str(err)`

Reply:
I think it is probably different.

Reply:
sounds good